### ISSUE TITLE
Npc's inventory doesn't clear over time

### ISSUE THREAD

**Issue by eric-ycw** [ironbane/IronbaneServerLegacy#466]
Vendors' inventory will still keep the items you sold to them, and will stay there until you buy it or restart the server.

**Comment by bsparks** [comment]
well not exactly a bug, it's currently programmed to do this. I think there are some other idea threads about ways to improve vendors and keep something similar to this behavior. maybe a google doc too

**Comment by ghost** [comment]
Yeah I believe it was done to fix something else. But both occurrences were annoying. We do have designs in the Google Docs, though. I believe I did come up with a system. I think everyone commented on it but no one started working on it, hahah.
